_posts/2024-02-28-How-to-contribute-to-WildFly.adoc

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title:  "How to contribute to WildFly"
+date:   2024-02-28
+tags:   WildFly contribution
+author: rchakrab
+description: Article about How to contribute to WildFly.
+---
+
+= How to contribute to WildFly
+
+Welcome to the world of WildFly contributions, a place where your skills can make a real impact! If you're eager to dive in but not sure where to start, this guide is here to simplify the process for you. Whether you're new to open-source or just getting acquainted with WildFly, we've got your back.
+
+In the upcoming sections, we'll walk you through the essential steps, providing insights into the WildFly ecosystem and equipping you with the tools and knowledge needed to actively participate in its development. From setting up your development environment to engaging with the vibrant community, each step is a gateway to not only enhancing your technical skills but also fostering connections within a community passionate about open-source excellence. Join us as we explore how you can navigate the wild and make your mark in the world of WildFly contributions.
+
+== Get Acquainted with WildFly:
+
+The first step in contributing to WildFly is to familiarize yourself with the project. Visit the WildFly https://github.com/wildfly/wildfly[GitHub repository]. Explore the project structure, review the source code, and read the https://docs.wildfly.org/[latest documentation] available in the repository. The documentation provides valuable insights into building, configuring, and deploying applications with WildFly.
+
+== Set Up Your Development Environment and Install WildFly:
+
+Before installing the latest version of WildFly, there are a few prerequisites:
+
+* Check that you have a JDK on your machine—JDK 11 or higher recommended to start WildFly. You can use the open source JDK called OpenJDK. Once you install the JDK, set the JAVA_HOME environment variable.
+
+* Ensure you have Maven 3.6.0 or higher installed. You can download Maven from here and set the environment variables.
+
+* After loading both the variables, check the versions of JDK and Maven.
+
+There are many ways you can install WildFly, including unzipping our traditional download zip, provisioning a custom installation using Galleon, or building a bootable jar. The official installation guide helps you identify the kind of WildFly installation that best fits your application’s deployment needs.
+
+== Create an Issue or Follow the JIRA:
+
+WildFly uses JIRA for issue tracking. You can find the list of issues and tasks on the https://issues.redhat.com/projects/WFLY[WildFly JIRA dashboard]. If you encounter a bug, have a feature request, or want to contribute in any way, create a new issue on JIRA. Alternatively, you can pick an existing issue or go through https://issues.redhat.com/browse/WFLY-18723?jql=project%20%3D%20WFLY%20AND%20labels%20%3D%20%22good-first-issue%22%20AND%20resolution%20%3D%20Unresolved[good first issues] that aligns with your interests or skills.
```

Comment:
"If you encounter a bug, have a feature request,..."

Maybe we could also mention something like "spot something in the documentation that could be improved or spot something missing in the documentation" since it's also good to have JIRAs created for this too.